services/order.py

```diff
@@ -0,0 +1,63 @@
+
+from datetime import datetime
+from django.utils import timezone
+from django.db import transaction
+from django.core.exceptions import ValidationError
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    # Знаходимо користувача за username
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        raise ValidationError(
+            f"User with username '{username}' does not exist"
+        )
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+        if date:
+            try:
+                order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+            except ValueError:
+                raise ValidationError(
+                    "Invalid date formate. Use 'YYYY-MM-DD HH:MM'."
+                )
+        else:
+            order.created_at = timezone.now()
+
+        order.save()
```

Comment:
According to the checklist, avoid calling `save()` multiple times if possible. Consider setting all attributes before saving the order.